burgle.lic

```diff
@@ -477,7 +477,7 @@ class Burgle
 
   #ripped out of steal.lic
   def drop_item(item)
-    case bput("drop my #{item}", 'You drop', 'You wince', 'would damage it', 'smashing it to bits', 'Something appears different about', 'What were you')
+    case bput("drop my #{item}", 'You drop', 'You spread', 'You wince', 'would damage it', 'smashing it to bits', 'Something appears different about', 'What were you')
```

Comment:
What do you think about using `DRCI.dispose_trash` here?

Reply:
Yes, that will be a good change. Before we can do that swap we'll need to add some match strings that are used here that aren't yet used by `DRCI.dispose_trash`, like "You wince", "smashing it to bits", etc.

I'll get another PR for `common-items` submitted that'll need to get merged before we refactor this method to use DRCI.dispose_trash.

In the interim if you're ok with it, the "You spread" change could go ahead and be merged and the PR to switch to DRCI would chase it.